### PR TITLE
accordion and accordion item unit tests

### PR DESCRIPTION
**Pull request by brianchristopherbrady**
Adds `accordion` and `accordion` item unit tests

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/c83bfec16fd0d93b1305e5f5aec5d8a62c8f7cb7'>c83bfec16fd0d93b1305e5f5aec5d8a62c8f7cb7</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Review by procload — CHANGES_REQUESTED** [review]
Looking good! There are a few tests we might want to add, however:
* Do we want to test keyboard navigation? Specifically moving between the elements and expanding/collapsing with the Enter key?
* How can we test whether or not the items are expandable (or not) when disabled? [Check out what I'm doing for the radio group here](https://github.com/microsoft/fluentui/pull/28826/files#diff-284c1170817764783c78596674612e5b50dae8dbd894259d9dcc219f13c929d5R293-R342).
* We should test for the ability to add custom icons and presentation icons. I did something [similar on the CounterBadge component here](https://github.com/microsoft/fluentui/pull/28867/files#diff-1ce40f3696e610e18f1d8ec650c19eac6d36fda965af36496404995d963c428aR24-R81).

**Review by procload — CHANGES_REQUESTED** [review]
Looking good! There are a few tests we might want to add, however:
* Do we want to test keyboard navigation? Specifically moving between the elements and expanding/collapsing with the Enter key?
* How can we test whether or not the items are expandable (or not) when disabled? [Check out what I'm doing for the radio group here](https://github.com/microsoft/fluentui/pull/28826/files#diff-284c1170817764783c78596674612e5b50dae8dbd894259d9dcc219f13c929d5R293-R342).
* We should test for the ability to add custom icons and presentation icons. I did something [similar on the CounterBadge component here](https://github.com/microsoft/fluentui/pull/28867/files#diff-1ce40f3696e610e18f1d8ec650c19eac6d36fda965af36496404995d963c428aR24-R81).

File: packages/web-components/src/accordion/accordion.spec.ts [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+
+    await expect(firstItem).not.toHaveAttribute('expanded', '');
+
+    await expect(secondItem).toHaveAttribute('expanded', '');
+
+    await expect(thirdItem).not.toHaveAttribute('expanded', '');
+  });
+
+  test('should allow disabled items to be expanded when in single mode', async () => {
+    test.slow();
```
[the remark this answers is not in the document]
Comment (halerankin) [reply packages/web-components/src/accordion/accordion.spec.ts:293]: Two items expanded while in single expand-mode? Is there a use case for this in the Figma?

**Review by halerankin — APPROVED** [review]
w/ comment

**Review by procload — APPROVED** [review]

**Review by miroslavstastny — APPROVED** [review]

**Pull request merged**